src/ifcgeom/IfcGeomIterator.h

```diff
@@ -317,6 +317,8 @@ namespace IfcGeom {
 
 		IfcParse::IfcFile* getFile() const { return ifc_file; }
 
+		Kernel* getKernel() { return &kernel; }
```

Comment:
The `getKernel()` method breaks the SWIG Python wrapper, because SWIG is not aware of the `Kernel` type. If we need to keep it, you can ignore the function prototype in the interface file [1].

[1] https://github.com/IfcOpenShell/IfcOpenShell/blob/master/src/ifcwrap/IfcGeomWrapper.i#L117

~~~
/home/travis/build/IfcOpenShell/IfcOpenShell/cmake/build-ifc2x3/ifcwrap/IfcPythonPYTHON_wrap.cxx: In function ‘PyObject* _wrap_iterator_single_precision_getKernel(PyObject*, PyObject*)’:

/home/travis/build/IfcOpenShell/IfcOpenShell/cmake/build-ifc2x3/ifcwrap/IfcPythonPYTHON_wrap.cxx:6242:3: error: ‘Kernel’ was not declared in this scope

   Kernel *result = 0 ;
~~~